### ISSUE TITLE
Server error on G2C data

### ISSUE THREAD

**Issue by roed314** [LMFDB/lmfdb#5068]
From the flasklog:
```
Exception on /Genus2Curve/Q/data/  [GET]
Traceback (most recent call last):
  File "/home/sage/sage-9.4/local/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/sage/sage-9.4/local/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/sage/sage-9.4/local/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/sage/sage-9.4/local/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/sage/sage-9.4/local/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/sage/sage-9.4/local/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/lmfdb/lmfdb-git-web/lmfdb/genus2_curves/main.py", line 391, in G2C_data
    bread = get_bread([(label, url_for_curve_label(label)), ("Data", " ")])
  File "/home/lmfdb/lmfdb-git-web/lmfdb/genus2_curves/main.py", line 375, in url_for_curve_label
    alpha=slabel[1],
IndexError: list index out of range
[2022-03-09 09:42:45 UTC] 500 error on URL https://www.lmfdb.org/Genus2Curve/Q/data/%20
```

**Comment by AndrewVSutherland** [comment]
I'm not able to reproduce this error, can you provide a URL that does?

**Comment by edgarcosta** [comment]
https://www.lmfdb.org/Genus2Curve/Q/data/%20

**Comment by AndrewVSutherland** [comment]
Thanks, I was actually using a space rather than %20.  I note that https://www.lmfdb.org/Genus2Curve/Q/data/banana also works.

**Comment by AndrewVSutherland** [comment]
This issue is present on some other routes as well, including:
- https://www.lmfdb.org/Character/Dirichlet/data/banana
- https://www.lmfdb.org/EllipticCurve/data/banana
- https://www.lmfdb.org/ModularForm/GL2/TotallyReal/data/banana

**Comment by AndrewVSutherland** [comment]
Fixed via #5076
